python/equistore-operations/equistore/operations/_dispatch.py

```diff
@@ -99,11 +101,13 @@ def bincount(input, weights=None, minlength=0):
         raise TypeError(UNKNOWN_ARRAY_TYPE)
 
 
-def list_to_array(array, data: List):
-    """Create an object from data with the same type as ``array``."""
+def array_like_data(array, data):
```

Comment:
this was changed in #319, why do we need to changed it back?

Reply:
Yes, sorry, I pasted a whole file that wasn't up to date